ci/infra/azure/terraform.tfvars.example

```diff
@@ -0,0 +1,117 @@
+# AZURE location
+# A list of location names can be found here:
+# https://azure.microsoft.com/en-in/global-infrastructure/locations/
+# https://azure.microsoft.com/en-us/global-infrastructure/regions/
+# https://docs.microsoft.com/en-us/azure/availability-zones/az-region
+# azure_location = "West Europe"
+azure_location = "East Asia"
+
+# Enable zone for available regions
+enable_zone = false
```

Comment:
I would turn it on inside of the example

Reply:
changed